pkg/system/chmod.go

```diff
@@ -0,0 +1,17 @@
+package system
+
+import (
+	"errors"
+	"os"
+	"syscall"
+)
+
+func Chmod(name string, mode os.FileMode) error {
+	err := os.Chmod(name, mode)
+
+	for err != nil && errors.Is(err, syscall.EINTR) {
+		err = os.Chmod(name, mode)
+	}
```

Comment:
why not continually retry? is only one recall sufficient?

Reply:
lol I read for as if. nevermind :upside_down_face:

Reply:
So glad I'm not the only one to have done that!

Reply:
It's probably because `for` is golang's `while`.